polus-notebook-plugin/src/execute_notebook.py

```diff
@@ -14,36 +22,47 @@ def main():
     parser.add_argument('--input-notebook', dest='input_notebook', type=str, help='Jupyter notebook to run', required=True)
     parser.add_argument('--output-collection', dest='output_collection', type=str, help='output collection', required=True)
     parser.add_argument('--output-notebook', dest='output_notebook', type=str, help='executed notebook', required=True)
+    parser.add_argument('--config-file', dest='config_file', type=str, help='configuration file', required=False)
     args = parser.parse_args()
     
     input_collection = args.input_collection
     input_notebook = os.path.join(args.input_notebook, 'notebook.ipynb')
     output_collection = args.output_collection
     output_notebook = os.path.join(args.output_notebook, 'notebook.ipynb')
+    config_file=args.config_file
```

Comment:
Don't forget to check for subfolders here. If the folder passed to this plugin is the result of another plugin, then you will have to switch to the `metadata_files` folder.

Reply:
The plugin provides the image collection path containing the config file to the notebook. Changes such as switching to the `metadata_files` directory should be done within the notebook.